plugin/trino-password-authenticators/src/test/java/io/trino/plugin/password/ldap/TestLdapAuthenticator.java

```diff
@@ -31,16 +38,30 @@
     private final Closer closer = Closer.create();
 
     private TestingOpenLdapServer openLdapServer;
+    private String proxyLdapUrl;
     private LdapAuthenticatorClient client;
 
     @BeforeClass
     public void setup()
             throws Exception
     {
-        openLdapServer = new TestingOpenLdapServer();
+        Network network = Network.newNetwork();
+        closer.register(network::close);
+
+        ToxiproxyContainer proxyServer = new ToxiproxyContainer("shopify/toxiproxy:2.1.0")
```

Comment:
Please have separate class to that plays with containers network. I would like to not affect other tests.